src/test/java/com/github/jscookie/javacookie/test/unit/CookiesDecodingTest.java

```diff
@@ -27,11 +27,27 @@ public void character_not_allowed_in_name_and_value() {
 		Assert.assertEquals( expected, actual );
 	}
 
+	@Test
+	public void character_with_2_bytes() {
+		Mockito.when( request.getHeader( "cookie" ) ).thenReturn( "c=%C3%A3" );
+		String actual = cookies.get( "c" );
+		String expected = "ã";
+		Assert.assertEquals( expected, actual );
+	}
```

Comment:
This test case is failing.., (its also failing in Travis CI builds - https://travis-ci.org/github/js-cookie/java-cookie/builds/764105675)
![java-cookie pr reivew faing junit](https://user-images.githubusercontent.com/13982269/118262249-3f1e1500-b4d2-11eb-9468-848884d3f822.JPG)

@tholu check this failing test case

Reply:
@JevinMenezes Yes, I explicitly added the test case here: https://github.com/js-cookie/java-cookie/pull/21/commits/88b832788c9e5fc46ba13da2db2834bbcff199ec

Thanks for the reminder! Did not have time yet to fix the issue, will try to get this done over the weekend.